src/content-modals/TNCModal.vue

```diff
@@ -96,10 +96,10 @@
     <h4>5. Judging</h4>
     <Para>
       a. At the end of the 24-hour hacking period, the Participants will present their Submissions through a platform
-      that is to be determined by the Organisers and will be given at least 30 minutes (any request for additional time
-      may be granted at the sole discretion of the Organiser) to set up their presentation booths. Each Participant or
-      team with a Submission will be given up to four (4) minutes to present his/her/their Submission to a panel of
-      judges. The order of the Participants’ or teams’ presentations will be determined by the Organiser.
+      that is to be determined by the Organisers. Each Participant or Team with a Submission will be given up to 20
+      minutes (any request for additional time may be granted at the sole discretion of the Organiser) to set up their
+      virtual presentation booths , and up to four (4) minutes to present his/her/their Submission to a panel of judges.
```

Comment:
Hi @FolkLoreee! For this change:

```diff
+ virtual presentation booths , and ...
```

Could you remove the whitespace before the comma, so that it becomes this:

```diff
+ virtual presentation booths, and ...
```

Other than that, the rest LGTM!

Reply:
@jamestiotio oki, I just updated it. thanks!

Reply:
Oki nice, thanks boss